config/v1/types_build.go

```diff
@@ -72,6 +76,27 @@ type ImageLabel struct {
 	Value string `json:"value,omitempty"`
 }
 
+type RegistriesList []string
+
+type RegistriesConfig struct {
+	// SearchRegistries lists the registries to search for images if an image repository is not specified in an image pull spec
+	// +optional
+	SearchRegistries *RegistriesList `json:"searchRegistries,omitempty"`
```

Comment:
Builds will default to using `docker.io` for the search registry. Pointer here allows cluster admins to override this behavior. See https://github.com/openshift/builder/pull/36

Others use pointers for API consistency.

Reply:
I don't think making the rest pointers for consistency(when we don't otherwise have a good reason to make them pointers) is appropriate but i'll let @openshift/api-reviewers make that call.

Reply:
> Builds will default to using `docker.io` for the search registry. Pointer here allows cluster admins to override this behavior. See [openshift/builder#36](https://github.com/openshift/builder/pull/36)

This implies that nil and empty slice have different behavior.  Please document the difference.

Reply:
> I don't think making the rest pointers for consistency(when we don't otherwise have a good reason to make them pointers) is appropriate but i'll let @openshift/api-reviewers make that call.

Correct.  Only things that must be pointers should be pointers and the difference in behavior should be clearly documented